modMcf/src/org/aion/mcf/vm/types/KernelInterfaceForFastVM.java

```diff
@@ -121,6 +127,16 @@ public void deductEnergyCost(Address address, BigInteger energyCost) {
         }
     }
 
+    @Override
+    public void refundAccount(Address address, BigInteger amount) {
+        // TODO: remove this placeholder for changes made in a reorganized commit
+    }
+
+    @Override
+    public void payMiningFee(Address miner, BigInteger fee) {
+        // TODO: remove this placeholder for changes made in a reorganized commit
```

Comment:
was this TODO forgotten about?  or is it for a later commit to fix?

Reply:
same as above